fl4health/utils/functions.py

```diff
@@ -1,8 +1,30 @@
+from logging import WARNING
+from typing import Tuple
+
 import torch.nn as nn
+from flwr.common.logger import log
 from flwr.common.parameter import ndarrays_to_parameters
 from flwr.common.typing import Parameters
+from opacus.validators import ModuleValidator
 
 
 def get_all_model_parameters(model: nn.Module) -> Parameters:
     # Extracting all model parameters and converting to Parameters object
     return ndarrays_to_parameters([val.cpu().numpy() for _, val in model.state_dict().items()])
+
+
+def privacy_validate_and_fix_modules(model: nn.Module) -> Tuple[nn.Module, bool]:
```

Comment:
nit: Not a huge deal at all, but I don't love the naming on this file (functions.py). I think we should avoid having catch alls where we accumulate vaguely related functions. Its not super intuitive where to find things.

Reply:
Lol...that's fair. I'll put something in the backlog to change this, since it also means we probably want to move `get_all_model_parameters`

Reply:
Added to the backlog